modelina-website/src/components/playground/options/JavaGeneratorOptions.tsx

```diff
@@ -173,8 +213,15 @@ class JavaGeneratorOptions extends React.Component<
             />
           </label>
         </li>
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
+        <li className=' flex items-center'>
+          <InfoModal text="Include Overwrite Equal Support :">
+            <p>
+            n Java, the equals() method is used to determine if two objects are equal based on their content rather than their memory addresses. The default implementation of equals() in the Object class performs a reference equality check, meaning it only returns true if the compared objects are the same instance in memory.
+            <br/><br/>
+            By enabling the option to overwrite equal support, you can provide your own implementation of the equals() method in a class. This allows you to define the equality criteria based on the specific properties and requirements of the class.
```

Comment:
This is very generic wording about what `toString` is, however it does not explain what it does to the code generation.

Reply:
fixed!! :tada: